netci.groovy

```diff
@@ -32,4 +32,55 @@ def branch = GithubBranchName
             Utilities.addGithubPushTrigger(newJob)
         }
     }
-}
\ No newline at end of file
+}
+
+// Add VSI job.
+// For now, trigger VSI jobs only on explicit request.
+def newVsiJobName = Utilities.getFullJobName(project, "vsi", false /* isPr */)
+
+def newVsiJob = job(newVsiJobName) {
+    description('')
+    label('windows-roslyn-internal-dev15-project-system')
+
+    // This opens the set of build steps that will be run.
+    steps {
+        // Build roslyn-project-system repo.
+        batchFile("build.cmd /release")
+
+        // git clone roslyn-internal to build and run VSI tao tests.
+        batchFile("""git clone https://github.com/dotnet/roslyn-internal.git
+pushd roslyn-internal
+git submodule init
+git submodule sync
+git submodule update --init --recursive
```

Comment:
You may be correct, though I have kept this consistent with https://github.com/dotnet/roslyn-internal/blob/master/netci.groovy#L114